test/models/fake_auth_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'test_helper'
+
+class FakeAuthTest < ActiveSupport::TestCase
+  include FakeAuthConfig
+
+  test 'fakeauth disabled' do
+    ClimateControl.modify(
+      FAKE_AUTH_ENABLED: 'false',
+      HEROKU_APP_NAME: 'thesis-submit-pr-123'
```

Comment:
Probably better to use a tacos pattern to confirm specifically that `FAKE_AUTH_ENABLED` being false is enough to shut it off even if `HEROKU_APP_NAME` would be allowed for these two assertions.

Reply:
What, you're criticizing my copypasta? 😅